test/extended/networking/network_segmentation.go

```diff
@@ -125,7 +124,7 @@ var _ = Describe("[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:User
 					Entry(
 						"for two pods connected over a L2 primary UDN",
 						networkAttachmentConfigParams{
-							name:     nadName,
+							name:     randomNetworkMetaName(),
```

Comment:
there's no point doing this for NADs, since they already randomize the `networkName` attribute - which is the ID of the network.


for UDNs (cluster or not) I think you *do* need to randomize.

Reply:
Fixed, following discussion at https://github.com/ovn-kubernetes/ovn-kubernetes/pull/4842#discussion_r1866845321